macros/edr/data_monitoring/monitors_query/column_monitoring_query.sql

```diff
@@ -72,6 +72,8 @@
             metric_name,
             {{ elementary.cast_as_float('metric_value') }} as metric_value,
             {{ elementary.null_string() }} as source_value,
+            {{ elementary.null_string() }} as dimension,
+            {{ elementary.null_string() }} as dimension_value,
```

Comment:
I prefer to be consistent, if at the final CTE these columns are last, they should be last here as well.
It will prevent future mistakes when we make changes.